lib/collapsers/binary.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+var logger = require('bole')('collapsify:collapsers:binary');
+var base64Utils = require('../utils/base-64');
+
+function external(url, opts) {
+  logger.info('Fetching external binary from "%s".', url);
+
+  return opts.fetch(url).then(collapse);
```

Comment:
Looks like we're doing this `opts.fetch` in each plugin, can we move that out to the lib itself so that we would avoid repetitive functions?